### DISCUSSION TITLE
Question about Nefarious Essence

### DISCUSSION THREAD

**Discussion by PrimordialDoom** [Aizistral-Studios/Enigmatic-Legacy#309] [Q&A]
Couldn't find any info on the wiki or from anywhere online about the item. Where is one supposed to get this? I put it on to try it out in my friends server but it'd very underwhelming and I guess meant to be used late game or if you hate having fun. But I wanted to craft the twisted heart but there is insufficient information on the essence required to create it.

**Comment by PrimordialDoom** [comment]
I just want to get rid of the ring without having to use creative if possible, already spoiled the modpack I was on by trying to find it in a single player creative world with no success.

**Comment by ambientNexus** [comment]


**Reply by PrimordialDoom** [reply]
> You have to kill the wither with the ring equipped. Seriously, did you even bother reading the tooltip?

It's rude to assume that I wouldn't even bother reading the tooltip. Obviously I tried killing the Wither in a creative test world with the ring equipped. Didn't drop the item.

**Reply by Aizistral** [reply]
If you can reproduce this outside of the modpack, submit a bug report: https://github.com/Aizistral-Studios/Enigmatic-Legacy/issues
Otherwise contact modpack developer(s).

**Reply by PrimordialDoom** [reply]
> If you can reproduce this outside of the modpack, submit a bug report: https://github.com/Aizistral-Studios/Enigmatic-Legacy/issues Otherwise contact modpack developer(s).

Alright, but thank you for trying to help so far, I will try to ask the modpack developer(s) through their discord. Have a good day :D

**Comment by Aizistral** [comment]
> But I wanted to craft the twisted heart but there is insufficient information on the essence required to create it

By default Twisted Heart does not require Nefarious Essence to be crafted, it's a change from the modpack itself, which I have no influence over.

> I just want to get rid of the ring without having to use creative if possible, already spoiled the modpack I was on by trying to find it in a single player creative world with no success.

The only way to do that without creative mode is using Unholy Stone. Its tooltip pretty clearly describes how to use it once you have it.

**Reply by PrimordialDoom** [reply]
Thanks for the reply, but I thought you needed the Twisted Heart to craft the Unholy Stone. And I did read the tooltip, but thanks for reiterating it kindly. And if it is a modpack change itself (Another Quality Mod Pack 3) then I understand it may be out of your control or know what to do for my situation.


The modpack showed the Twisted Heart recipe requiring the Nefarious essence and killing Wither Skeletons and The Wither did not cause it to drop with the ring. Not sure if Creative Mode would prevent it from dropping or if I needed to do something else to get it to drop. I checked multiple generating loot tables as well to no avail, so I was just stumped.
